front-end/src/components/DropDownComponent/DropDown.tsx

```diff
@@ -0,0 +1,80 @@
+import React, {useEffect, useRef, useState} from 'react'
+import './DropDown.css'
+
+interface DropDownProps {
+  elements: String[]
+  selectedElement: String
```

Comment:
```suggestion
  elements: string[]
  selectedElement: string
```
String is JavaScript type for string,
string is TypeScript type for string.

Reply:
fixed in the next commit